src/subscription/subscription.service.ts

```diff
@@ -49,18 +49,18 @@ export class SubscriptionService {
     }
 
     async findAll(): Promise<Subscription[]> {
-        const subscripionsResult = await this.subscriptionRepository.find();
-        return subscripionsResult;
+        const subscriptionsResult = await this.subscriptionRepository.find();
+        return subscriptionsResult;
```

Comment:
Could this be simplified down to `return await await this.subscriptionRepository.find()` ?

Reply:
Good call, will change it to do that!